pkg/crawler/crawler.go

```diff
@@ -58,8 +57,12 @@ func NewCrawler(opt Option) Crawler {
 		}
 	}
 	client.ErrorHandler = func(resp *http.Response, err error, numTries int) (*http.Response, error) {
-		logger := slog.With(slog.String("url", resp.Request.URL.String()), slog.Int("status_code", resp.StatusCode),
-			slog.Int("num_tries", numTries))
+		logger := &slog.Logger{}
```

Comment:
If we create a new logger, the default setting is not respected.
https://github.com/aquasecurity/trivy-java-db/blob/577c6a3e4afcab91ef54d8a03709578acb6e1948/cmd/trivy-java-db/main.go#L65

```suggestion
		logger := slog.Default()
```

Reply:
nice catch! Updated in https://github.com/aquasecurity/trivy-java-db/pull/42/commits/111f5bb7b7870544effa9cf717ceaf472115357f